code/modules/mob/living/silicon/death.dm

```diff
@@ -6,3 +6,9 @@
 
 /mob/living/silicon/spawn_dust()
 	new /obj/effect/decal/remains/robot(loc)
+
+/mob/living/silicon/death(gibbed)
+	diag_hud_set_status()
+	diag_hud_set_health()
+	update_health_hud()
+	..()
```

Comment:
Does this not need a return `..(gibbed)` or whatever? I do not really know, it just looks to me like it would.